Error in maker.makecode.com after loading library

imulator ServiceWorker registration successful with scope:  https://trg-maker.userpxt.io/
pxtapp.js:1 webusb: get devices
pxtapp.js:1 error: pxt_modules/ws2812b/main.ts(3,50): error TS2304: Cannot find name 'DigitalPin'.
error: pxt_modules/ws2812b/main.ts(7,40): error TS2304: Cannot find name 'DigitalPin'.

5pxtapp.js:1 webusb: get devices
pxtapp.js:1 error: pxt_modules/ws2812b/main.ts(3,50): error TS2304: Cannot find name 'DigitalPin'.
error: pxt_modules/ws2812b/main.ts(7,40): error TS2304: Cannot find name 'DigitalPin'.

--- a/neopixel.ts
+++ b/neopixel.ts
@@ -615,3 +615,10 @@ namespace neopixel {
         Shortest
     }
 }
+
+namespace ws2812b {
+    /**
+     * DigitalPin interface to satisfy the compiler in targets where it is not defined.
+     */
+    export interface DigitalPin { }
+}

